tests/ssh-hassh/test.yaml

```diff
@@ -0,0 +1,20 @@
+requires:
+  min-version: 5.0.0
+features:
+    - RUST
+
+args:
+ - -k none
+
+checks:
+  # Check that we have the following events in eve.json
+  # 2 - alert events
```

Comment:
The comment says 2 and the code says 4...

Reply:
Sorry, my bad.